### PR TITLE
Feat: Add tv 

### PR DESCRIPTION
**Pull request by Chrinsieboy**

### CODE REVIEW

File: src/components/AddTvForm.vue [hunk 1]
```diff
@@ -4,15 +4,20 @@
       <div class="flex justify-center w-[90rem] p-2 rounded-md bg-gray-200 drop-shadow-md">
           <div class="w-full p-5 text-left">
             <h1 class="mb-2 text-5xl font-medium text-center">Add TV</h1>
-            <form action="" method="POST">
+              <div class="bg-primary" v-if="isSuccess">
+                <h1 class="m-2 text-lg text-center text-white">TV Succesfull add</h1>
```
Comment (martvdm) [inline src/components/AddTvForm.vue:8]: successful* (Typo)

**Review by DanericNetwork — APPROVED** [review]

File: src/components/AddTvForm.vue [hunk 2]
```diff
@@ -4,15 +4,20 @@
...
+              </div>
+              <div class="bg-quaternary" v-if="isFail">
+                <h1 class="m-2 text-lg text-center text-white">TV was not added</h1>
+              </div>
                 <div class="flex flex-row justify-evenly">
                     <div class="flex flex-col mx-1 mb-3 sm:mx-5">
-                        <label for="username" class="text-sm font-medium">TV Name:<br>
-                        <input type="text" id="username" class="p-2 mt-[2px] font-semibold border-2 border-secondary rounded-md w-96" placeholder="TV Name">
+                        <label for="name" class="text-sm font-medium">TV Name:<br>
+                        <input v-model="name" type="text" id="name" class="p-2 mt-[2px] font-semibold border-2 border-secondary rounded-md w-96" placeholder="TV Name">
```
Comment (martvdm) [inline src/components/AddTvForm.vue:16]: id name?

**Review by JustRutger — APPROVED** [review]
🚀🚀🚀

**Review by martvdm — APPROVED** [review]

File: src/components/AddTvForm.vue (hunk 1 above)
Comment (Chrinsieboy) [reply src/components/AddTvForm.vue]: Fixed

File: src/components/AddTvForm.vue (hunk 2 above)
Comment (Chrinsieboy) [reply src/components/AddTvForm.vue]: got rid of it

**Review by martvdm — APPROVED** [review]

**Pull request merged**